python/ray/tune/utils/callback.py

```diff
@@ -45,7 +45,7 @@ def _create_default_callbacks(
     sync_config: SyncConfig,
     air_verbosity: Optional["AirVerbosity"] = None,
     metric: Optional[str] = None,
-    progress_metrics: Optional[List[str]] = None,
+    progress_metrics: Optional[Tuple] = None,
```

Comment:
Nit: There is no Args list in the docstring that allows me to check, what format e.g. `progress_metrics` should have and what the items in the Tuple should be like.

Reply:
Updated to `Collection[str]`!